client/app/bundles/App/GettingStarted/Theming.md

```diff
@@ -1,3 +1,103 @@
 ## Theming
```

Comment:
Nice documentation & explanation on the theming configuration.

Reply:
ty